apps/studio/src/components/tableview/TableTable.vue

```diff
@@ -614,6 +614,11 @@ export default Vue.extend({
           headerTooltip += ' [Primary Key]'
         }
 
+        // if column has a comment, add it to the tooltip
+        if (column.comment) {
+          headerTooltip += `<br/> ${column.comment}`
```

Comment:
This needs to be made HTML safe, but otherwise, love this! So:

```
headerTooltip += `<br/> ${escapeHtml(column.comment)}`

```

Once you've made that change I will merge!